### ISSUE TITLE
Include automated test for ensuring all fields include labels

### ISSUE THREAD

**Issue by danielbachhuber** [WordPress/gutenberg#6098]
For accessibility purposes, all Gutenberg fields should include labels. Because it's easy for humans to forget this, it would be ideal to include some automated test to force this.

cc @afercia 

From https://github.com/WordPress/gutenberg/pull/6033#issuecomment-380099234

**Comment by afercia** [comment]
Thanks @danielbachhuber this is something I'd definitely recommend to consider. See also empty and unlabelled buttons https://github.com/WordPress/gutenberg/issues/5823#issue-309049606
> Since it happened several times that completely unlabeled elements were introduced in the UI, would it be feasible to use some specific tests? Specifically for buttons, is there any easy way to add tests to check if the buttons have some text or some non-empty aria-label to not allow empty buttons?

**Comment by danielbachhuber** [comment]
Filing against the `Ideas` project for now, as this isn't critical for WordPress 5.0

**Comment by afercia** [comment]
@danielbachhuber sorry probably I'm missing something but I can't find it under the `Ideas` project.

**Comment by danielbachhuber** [comment]
@afercia It's "Awaiting Triage":

<img width="268" alt="image" src="https://user-images.githubusercontent.com/36432/40656402-1f2ff818-62f9-11e8-8c48-a8b40898500e.png">

<img width="641" alt="image" src="https://user-images.githubusercontent.com/36432/40656421-28cbec06-62f9-11e8-9a48-9f300b3ab5bd.png">
